keyboards/geonworks/frogmini/fms/fms.c

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2022 Gondolindrim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "fms.h"
+
+void i2c_init(void) {
+    setPinInput(B6); // Try releasing special pins for a short time
+    setPinInput(B7);
+    wait_ms(10); // Wait for the release to happen
+
+    palSetPadMode(GPIOB, 6, PAL_MODE_ALTERNATE(4) | PAL_STM32_OTYPE_OPENDRAIN | PAL_STM32_PUPDR_FLOATING); // Set B6 to I2C function
+    palSetPadMode(GPIOB, 7, PAL_MODE_ALTERNATE(4) | PAL_STM32_OTYPE_OPENDRAIN | PAL_STM32_PUPDR_FLOATING); // Set B7 to I2C function
+}
+
```

Comment:
Potentially same as `keyboards/geonworks/frogmini/fmh/fmh.c` and can be removed

Reply:
Committed in [467b990](https://github.com/qmk/qmk_firmware/pull/17765/commits/467b990b90bb8d4de64eebe48136352a12dfaf20)